packit_service/config.py

```diff
@@ -276,20 +276,24 @@ def create_issue_if_needed(
         title: str,
         message: str,
         comment_to_existing: Optional[str] = None,
+        add_packit_prefix: Optional[bool] = True,
     ) -> Optional[Issue]:
         # TODO: Improve filtering
         issues = project.get_issue_list()
         packit_title = f"[packit] {title}"
 
         for issue in issues:
             if packit_title in issue.title or title in issue.title:
+                logger.debug(f"Title of issue {issue.id} matches.")
```

Comment:
Just wondering about the condition

Since `packit_title = f"[packit] {title}"`, then `packit_title in issue.title` is a subset of  `title in issue.title` and  thus can be removed from the OR, i.e. it can be changed to just

```python
if title in issue.title:
```

can't it?